### PR TITLE
Using single-spa-layout to position main content.

### PR DESCRIPTION
**Pull request by internettrans**
This is the first use of single-spa-layout in a "real" app. The idea is that instead of requiring every app to position itself below the navbar, we can have a "main content" div defined by single-spa-layout that is pre-positioned.

### CODE REVIEW

File: src/index.ejs [hunk 1]
```diff
@@ -22,6 +22,19 @@
   <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/extras/named-exports.min.js"></script>
 </head>
 <body>
+  <single-spa-router>
+    <nav class="topnav">
```
Comment (filoxo) [inline src/index.ejs:26]: The [devtools selector](https://github.com/react-microfrontends/navbar/blob/a6839022281b557ce085a72ff4be128c133554c6/src/react-mf-navbar.js#L38) should be changed to match this as well probably. It wasn't working before anyway.

**Comment by filoxo** [comment]
Oh I just read the docs. Would it be possible to setup a 404 route in this example as well? I think that would be great to have.

File: src/index.ejs (hunk 1 above)
Comment (internettrans) [reply src/index.ejs:26]: Good catch - I will remove the devtools selector from the navbar app. Single-spa-layout uses `<div id="single-spa-application:name"></div>` for the application containers, which the devtools automatically pick up on.

In other words, this single-spa-layout config will result in the following DOM:

```html
<nav class="topnav">
  <div id="single-spa-application:@react-mf/navbar">
  </div>
</nav>
```

**Comment by internettrans** [comment]
Ah yes I'll add a 404 route - good idea.

File: src/index.ejs (hunk 1 above)
Comment (filoxo) [reply src/index.ejs:26]: Oh yea, that's even better!

**Comment by internettrans** [comment]
Updated with 404 route

**Review by filoxo — APPROVED** [review]
🙈

**Pull request merged**